controller/release.go

```diff
@@ -63,6 +63,13 @@ func (r *ReleaseRepo) Add(data interface{}) error {
 		release.ArtifactIDs = []string{release.LegacyArtifactID}
 	}
 
+	if _, ok := release.Env[""]; ok {
+		return ct.ValidationError{
+			Field:   "env",
+			Message: fmt.Sprintf("you can't create an env var with an empty key (tried to set \"\"=\"%s\")", release.env[""]),
```

Comment:
Update the `if` statement so that it assigns the value to a variable and then use it here (rather than using `release.Env[""]` again):

```go
if value, ok := release.Env[""]; ok {
	// ... can reference `value` here
}
```

You can also use `%q` rather than `\"%s\"` to double-quote the variable in the output string.